### ISSUE TITLE
[TERMINAL_PASS_RELEASED] SWARM-1605 — PR802 current-carrier MODEL-341 random-init identity

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1605]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1605
ROLE: CONTINUOUS_AUTONOMOUS_SCIENTIFIC_COMPLETION_CLOSER
STATUS: ACTIVE_INDEPENDENT_VERIFY_PENDING_POSTCLAIM_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
PREFERRED_LANE: #2 D01 — Model Architecture
PREFERRED_WORK_KIND: INTEGRATION_CONVERGENCE
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / D01 CURRENT-CARRIER QUALIFICATION
SWARM_LANE_KEY: AUDIT-A|PR802|MODEL341-RANDOM-INIT-IDENTITY|CURRENT-CARRIER-EXACTHEAD-18FB94B0|V1
LIVE_MAIN_AT_CLAIM: 58711afeadf7f29c107188cb15d1e73c728768bb
TARGET_PR: #802
TARGET_BRANCH: model341/20m-candidate-a-20260826
TARGET_HEAD: 18fb94b0c5fe94817628a5ac007db2e64fc71992
TARGET_EXACT_HEAD_CI: 34730287647=SUCCESS
CURRENT_INTEGRATION_OWNER: #1594 / lawful successor
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIMED_AT_UTC_PLUS_6_HOURS unless released sooner
PRODUCT_MUTATION: false
AUTHORITATIVE_PRODUCT_PR_AUDIT_VERDICT: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false

ROUTING / ROTATION:
- worker 1605 routes to permanent lane #2 D01 and work kind INTEGRATION_CONVERGENCE;
- central integration is lawfully occupied by #1594, so this worker does not race merge authority;
- the current canonical MODEL-341 carrier moved to exact head 18fb94b0... after the previous narrow D01 exact-head qualifications. Historical #1226 explicitly states head movement invalidates its old verdict; current runtime owner #1604 needs a truthful FRESH_START random-init identity boundary on the current carrier;
- exact preclaim open semantic search for CURRENT-CARRIER-EXACTHEAD-18FB94B0 returned no existing exact-head D01 owner. #1604 owns bounded-pilot Product repair, not this read-only current-carrier D01 qualification.

MISSION:
Independently qualify only the exact current PR802 carrier's MODEL-341 scratch/random-init model identity mechanics, without Product mutation or any training claim. Determine whether the D02 recovery merge that produced 18fb94b0... preserved the canonical D01 ModelSpec/InitSpec/model-construction bytes and random-init boundary needed by downstream FRESH_START verification.

OWNED READ-ONLY SURFACE AFTER ARBITRATION WIN:
- exact PR802 head/base/state and exact-head shared CI;
- configs/candidates/model341_20m_candidate_a.json exact bytes/blob and canonical ModelSpec/InitSpec identities;
- src/twelve_six/model.py exact bytes/blob, model construction, parameter accounting, tied embedding/LM head behavior and initialization path;
- exact current-head D01-relevant test/evidence lineage needed to reproduce ModelSpec SHA-256 fbff24d561a2818453554d58ca23fc6ace3303b078f1935a8576c4565bd92441, InitSpec SHA-256 86483c6df623e80cab2f73aba718863fce18af6fe3b12430c1348414d92b48a5 and parameter total 20,613,440;
- compare current D01 blob identities with prior qualified carrier head(s) to determine whether earlier empirical random-init evidence remains byte-applicable;
- verify canonical construction remains own random initialization with no foreign/pretrained/from_pretrained load path in the audited D01 surface;
- verify seed-before-model-construction semantics only where an already-canonical current-carrier execution helper provides that evidence.

NONOVERLAP / AVOID:
- no Product mutation to PR802, PR1336/#1604, PR1256, D03/D04/D05/D06/D07/D10;
- no audit or mutation of #1604's live model-start-state tensor verifier itself;
- no duplicate PR1593/#1603 checkpoint replay audit;
- no central merge/integration action while #1594 owns it;
- no tokenizer fit, optimizer/model update, learned-target run, final-test access, paid compute, foreign pretrained weights or external LLM/API corpus intelligence.

ACCEPTANCE:
1. mandatory postclaim direct OPEN issues + OPEN PR collections newest-first at 100/page reach exact empty sentinels; any earlier valid same-semantic exact-head claimant wins and SWARM-1605 releases before verdict;
2. late-bind PR802 exact head and CI before verdict; head movement invalidates this exact-head package;
3. reproduce canonical ModelSpec and InitSpec identities and independent parameter total 20,613,440 from exact current bytes;
4. prove current model construction remains scratch/random-init and does not admit a pretrained/foreign-weight constructor path on the audited surface;
5. determine whether current model/config D01 blobs are byte-identical to the previously qualified exact carrier, so older empirical seed reproducibility evidence is reused only when mechanically justified;
6. exact-head shared CI must remain terminal SUCCESS for PASS; otherwise BLOCKED/CHANGES_REQUIRED;
7. publish durable exact evidence to this issue, permanent AUDIT-A #13, PR802, and handoff to #1604/#1594; then release this semantic. This audit never authorizes optimized-target exposure or training.

TRUTH_BOUNDARY_AT_CLAIM:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
TERMINAL AUDIT VERDICT — `PASS_WITH_NOTES / PASS_FOR_CURRENT_HEAD_MODEL_IDENTITY_RANDOM_INIT_MECHANICS`

SWARM_PROTOCOL: SWARM-300-V2
SWARM_WORKER_ID: SWARM-1605
LANE_KEY: `AUDIT-A|PR802|MODEL341-RANDOM-INIT-IDENTITY|CURRENT-CARRIER-EXACTHEAD-18FB94B0|V1`
TARGET: PR #802 exact head `18fb94b0c5fe94817628a5ac007db2e64fc71992`
TARGET_BASE: `main@0dc1aefffb7ea91fe27ca22cce9b4b0782ca507d`
EXACT_HEAD_SHARED_CI: `34730287647 = SUCCESS` (preflight SUCCESS; locked-x86-64 secret scan + clean install/package smoke/full checks SUCCESS; locked-arm64 clean install/package smoke SUCCESS).

OWNERSHIP / ARBITRATION
- mandatory postclaim direct OPEN issue collection exhausted newest-first at 100/page: pages 1..11 nonempty, page 12 exact `[]`;
- mandatory direct OPEN PR collection: pages 1..5 nonempty, page 6 exact `[]`;
- final exact lane-key and broader exact-head semantic searches returned only #1605; no earlier live same-semantic exact-head claimant found.

INDEPENDENT CURRENT-BYTE FINDINGS
1. `configs/candidates/model341_20m_candidate_a.json` current blob = `69e3cbd5f5c83c9d3d529a2a6376db3055979c40`; declares `canonical_base=random_init`, expected parameters `20,613,440`, ModelSpec identity `fbff24d561a2818453554d58ca23fc6ace3303b078f1935a8576c4565bd92441`, InitSpec identity `86483c6df623e80cab2f73aba718863fce18af6fe3b12430c1348414d92b48a5`.
2. Independent canonical-JSON recomputation (`sort_keys=True`, compact separators, UTF-8, no NaN) reproduces both identity SHA-256 values exactly.
3. Independent parameter algebra is exactly `20,613,440`: token embedding `81,920` + 16 × [attention `245,760` + SwiGLU MLP `1,036,800` + block norms `640`] + final norm `320`; tied LM head adds 0.
4. `src/twelve_six/model.py` current blob = `d0823aa666883ddb5c445a438730043ef8b50ff1`. `TwelveSixDecoder` constructs project-owned Embedding/TransformerBlocks/RMSNorm/Linear head directly, applies its own `InitSpec` normal initialization, applies residual-branch scaling, ties embeddings only after init, and checks live trainable parameter count against `ModelSpec.parameter_count()`. No `from_pretrained` result exists in repository search; no foreign/pretrained constructor path is present in this audited D01 surface.
5. `src/twelve_six/training/single_gpu.py` current blob = `37e58afc3aa81e2c7ee6ca05330050b019b59225`; `seed_before_model_init()` rejects bool/negative seeds and seeds Python, NumPy, torch (+ CUDA when applicable) before caller scratch model construction.
6. Prior exact-head #1466 empirical scratch-init evidence (external-load monkeypatch / same-seed reproducibility / different-seed divergence) is component-exact applicable here: compare `61aa37b340565dd1ba791adc16ce430f9b17fbaf..18fb94b0c5fe94817628a5ac007db2e64fc71992` shows only `src/twelve_six/training/trainer.py` and `tests/test_trainer_resume_integrity.py` changed; candidate config, model constructor, single-GPU seed helper and their passed component bytes did not change. PR #1543 likewise changed only those D02 resume files.
7. Important boundary: `Trainer.__init__` configures deterministic RNG after receiving an already-created model, so Trainer itself is NOT evidence for pre-construction seeding. The authoritative scratch-start mechanic is the separate `seed_before_model_init()` helper plus downstream runtime binding. #1604 remains owner of the live FRESH_START state verifier; this audit does not certify that verifier's separate Product semantics.

NON-BLOCKING NOTE A802-N01 RETAINED
Generic type canonicalization outside the exact canonical MODEL-341 packet is not globally strict. Any launch authority must bind the exact externally expected ModelSpec/InitSpec identities rather than trust arbitrary caller-recomputed aliases.

SCIENTIFIC VERDICT
`PASS_WITH_NOTES` for this exact current-head D01 MODEL-341 model-identity/random-init mechanic only. Head movement invalidates this verdict and requires fresh qualification.

THIS DOES NOT AUTHORIZE TRAINING OR TARGET EXPOSURE.
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

Ownership semantic released after durable handoff. Current integration owner / downstream owners must consume this as a narrow carrier prerequisite only.
